task/thermal/Cargo.toml

```diff
@@ -35,8 +35,9 @@ build-i2c = { path = "../../build/i2c" }
 build-util = { path = "../../build/util" }
 
 [features]
-gimlet = ["drv-gimlet-seq-api", "h753"]
-sidecar = ["drv-sidecar-seq-api", "drv-transceivers-api", "h753"]
+gimlet = ["dep:drv-gimlet-seq-api", "h753"]
+sidecar = ["dep:drv-sidecar-seq-api", "dep:drv-transceivers-api", "h753"]
```

Comment:
nitpicky, but personally i would prefer to avoid changing unrelated code if not necessary.

Reply:
Weird, I have no idea how this happened. I'll revert.